tox.ini

```diff
@@ -380,6 +380,10 @@ deps =
     pycryptodome>=3.10.1
     cosmpy>=0.4.1,<0.5.0
     setuptools==59.6.0
+    certifi<=2022.6.15.1
+    google-api-python-client<=2.60.0
+    idna<=3.3
+    protobuf<=3.20.1
```

Comment:
first 3 not in Pipfile and last one doesn't match specification there
https://github.com/valory-xyz/open-aea/blob/b1b9afbc3a6c97e10a8ddab7e0a27af24fc056e3/Pipfile#L48